test/Sentry.AspNetCore.Tests/SentryMiddlewareTests.cs

```diff
@@ -479,5 +481,37 @@ public void PopulateScope_Sdk_ContainNameAndVersion()
             Assert.Equal(Constants.SdkName, scope.Sdk.Name);
             Assert.Equal(SentryMiddleware.NameAndVersion.Version, scope.Sdk.Version);
         }
+
+        [Fact]
+        public async Task InvokeAsync_DefaultOptions_CanNotFlush()
+        {
+            var sut = _fixture.GetSut();
+            await sut.InvokeAsync(_fixture.HttpContext);
+            
+            _fixture.Options.DiagnosticLogger.Log(SentryLevel.Debug, "No events to flush.");            
```

Comment:
The test cases make sense but you are not asserting anything here. You just invoke the `Log` method on `DiagnosticLogger`. You need to verify it somehow.
like:
_fixture.Hub.FlushAsync(Arg.Any<TimeSpan>()).DidNotReceive();
```

```

Reply:
make's sense! Will be changed to related test cases as well.